src/index.ts

```diff
@@ -85,36 +90,36 @@ export default function createClient<Paths extends {}>(options?: ClientOptions)
 
   return {
     /** Call a GET endpoint */
-    async get<Pathname extends PathsWith<Paths, 'get'>>(url: Pathname, init: FetchOptions<FilterKeys<Paths[Pathname], 'get'>>) {
-      return coreFetch(url, { ...init, method: 'GET' } as any);
+    async get<P extends PathsWith<Paths, 'get'>>(url: P, init: FetchOptions<FilterKeys<Paths[P], 'get'>>) {
+      return coreFetch<P, 'get'>(url, { ...init, method: 'GET' } as any);
```

Comment:
The issue causing #26 is that somehow, the type inference was “losing” the HTTP Method being used, and would filter _all_ methods under the same URL, which resulted in some confusion.

The “dumb” solution was just declare the URL and HTTP method statically here. Which isn’t a bad thing, because it means TS doesn’t have to do nearly as much work figuring out which URL is being used.